src/index.js

```diff
@@ -1,32 +1,36 @@
 // Load environment variables
-
 const dotenv = require("dotenv");
 dotenv.config({ path: ".env" });
 
+// Import Packages 
 const express = require("express");
 const morgan = require("morgan");
 const cors = require("cors");
+const bodyParser = require("body-parser");
 
+// Import Routes
 const router = require("./api");
 const { logger } = require("./utils/logger");
+const { postTestMessage } = require("./middleware/postTestMessage");
 const { errorHandler } = require("./middleware/error-handler");
 const { channelRouter } = require("./routes/channels/channels.router");
 const {
   previousPollsRouter
 } = require("./routes/previous-polls/previous-polls.router");
+const { newPollRouter } = require('./routes/new-poll/new-poll.router');
 const { resultRouter } = require("./routes/result/result.router");
 
-const { postTestMessage } = require("./middleware/postTestMessage");
+// Firebase Imports
 const Timestamp = require("firebase-admin").firestore.Timestamp;
 const db = require("./db/index");
 
+
 // Create a new express application instance
 const app = express();
 
 // The port the express app will listen on
 const port = process.env.PORT;
-
-logger.info("🤖 Initializing middleware");
+logger.info(":robot_face: Initializing middleware");
```

Comment:
lol does your terminal render this properly?

Reply:
Oh my. That's why the emojis are not showing.